Runtime/Core/ApiGatewayManagement/AmazonGameLiftClientWrapper.cs

```diff
@@ -1,35 +1,50 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Sockets;
 using System.Threading;
 using System.Threading.Tasks;
 using Amazon.GameLift;
 using Amazon.GameLift.Model;
+using AmazonGameLiftPlugin.Core.CredentialManagement;
+using AmazonGameLiftPlugin.Core.CredentialManagement.Models;
+using AmazonGameLiftPlugin.Core.Shared.FileSystem;
+using UnityEngine;
 
 namespace AmazonGameLiftPlugin.Core.ApiGatewayManagement
 {
     public class AmazonGameLiftClientWrapper : IAmazonGameLiftClientWrapper
     {
         private readonly IAmazonGameLift _amazonGameLiftClient;
-        private static readonly string s_dummyAwsAccessKey = "1";
-        private static readonly string s_dummyAwsSecretKey = "1";
-
+        private readonly ICredentialsStore _credentialsStore = new CredentialsStore(new FileWrapper());
+        private readonly GameLiftClientSettings _gameLiftClientSettings;
+        
+        /// <summary>
+        /// Client region is code dedicated to Amazon GameLift SDK calls made by the game client. 
+        /// </summary>
+        #region Client
+        
         internal AmazonGameLiftClientWrapper(IAmazonGameLift amazonGameLiftClient)
         {
             _amazonGameLiftClient = amazonGameLiftClient;
         }
 
-        public AmazonGameLiftClientWrapper(string localEndpoint)
+        public AmazonGameLiftClientWrapper(string profileName)
```

Comment:
~~Remove profileName, this argument isn't used.~~ Actually, remove this method as it's identical to the other constructor